Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py

```diff
@@ -271,14 +221,30 @@ def main():
         find_keywords(data, DBOT_TAG_FIELD, text_field, keyword_min_score)
     except Exception:
         pass
-
-    # evaluate model
-    human_readable, confusion_matrix, report = evaluate_model(train_text_data,
-                                                              train_tag_data,
-                                                              target_accuracy,
-                                                              max_samples_below_threshold,
-                                                              metric)
-
+    X = pd.Series(train_text_data)
+    y = pd.Series(train_tag_data)
+    n_splits = int(1.0 / (1 - TRAIN_SIZE_FOR_EVALUATION))
+    skf = StratifiedKFold(n_splits=n_splits, shuffle=False, random_state=None)
+    skf.get_n_splits(X, y)
+    train_index, test_index = list(skf.split(X, y))[-1]
+    X_train, X_test = list(X[train_index]), list(X[test_index])
+    y_train, y_test = list(y[train_index]), list(y[test_index])
+    model = demisto_ml.train_text_classifier(X_train, y_train)
+    ft_test_predictions = demisto_ml.predict(model, X_test, DEFAULT_LABEL_PREFIX)
```

Comment:
just call this function without the DEFAULT_LABEL_PREFIX - it has it

Reply:
Right, changed, thanks!